compute/compute-node.Dockerfile

```diff
@@ -1168,6 +1168,27 @@ RUN case "${PG_VERSION}" in \
     make BUILD_TYPE=release -j $(getconf _NPROCESSORS_ONLN) install && \
     echo 'trusted = true' >> /usr/local/pgsql/share/extension/pg_mooncake.control
 
+#########################################################################################
+#
+# Layer "pg_repack"
+# compile pg_repack extension
+#
+#########################################################################################
+
+FROM build-deps AS pg-repack-build
+ARG PG_VERSION
+COPY --from=pg-build /usr/local/pgsql/ /usr/local/pgsql/
+
+# pg_repack has not created a new tag with following commit, yet.
+ENV PG_REPACK_VERSION=85b64c6d4f599b2988343c4e7121acab505c9006
```

Comment:
Comment is obsolete now. And probably you should remove `PG_REPACK_VERSION` as well